crds/virtualmachines.yaml

```diff
@@ -0,0 +1,1660 @@
+                - blockDeviceRefs
+                - cpu
+                - memory
+                - virtualMachineClassName
+              type: object
+            status:
+              properties:
+                blockDeviceRefs:
+                  description: The list of attached block device attachments.
+                  items:
+                    properties:
+                      attached:
+                        description: The block device is attached to the virtual machine.
+                        type: boolean
+                      hotplugged:
+                        description: Block device is attached via hot plug connection.
+                        type: boolean
+                      kind:
+                        description: |-
+                          The BlockDeviceKind is a type of the block device. Options are:
+
+
+                          * `ClusterVirtualImage` — Use `ClusterVirtualImage` as the disk. This type is always mounted in RO mode. If the image is an iso-image, it will be mounted as a CDROM device.
+                          * `VirtualImage` — Use `VirtualImage` as the disk. This type is always mounted in RO mode. If the image is an iso-image, it will be mounted as a CDROM device.
+                          * `VirtualDisk` — Use `VirtualDisk` as the disk. This type is always mounted in RW mode.
+                        enum:
+                          - ClusterVirtualImage
+                          - VirtualImage
+                          - VirtualDisk
+                        type: string
+                      name:
+                        description: The name of attached resource.
+                        type: string
+                      size:
+                        description: The size of attached block device.
+                        type: string
+                      target:
+                        description: The name of attached block device.
+                        example: sda
+                        type: string
+                      virtualMachineBlockDeviceAttachmentName:
+                        description:
+                          The name of the `VirtualMachineBlockDeviceAttachment`
+                          resource that defines hot plug disk connection to the virtual
+                          machine.
+                        type: string
+                    required:
+                      - attached
+                      - kind
+                      - name
+                      - size
+                    type: object
+                  type: array
+                conditions:
+                  description: Specific points in VM's runtime.
+                  items:
+                    description:
+                      "Condition contains details for one aspect of the current
+                      state of this API Resource.\n---\nThis struct is intended for
+                      direct use as an array at the field path .status.conditions.  For
```

Comment:
do we really need this here?

Reply:
It was auto generated by kubebuilder. The imported type includes this description. Should I exclude this?